noxfile.py

```diff
@@ -234,7 +249,22 @@ def prerelease_deps(session):
         "pytest",
         "pytest-cov",
     )
-    session.install("-e", ".[all]")
+
+    with open(
+        CURRENT_DIRECTORY / "testing" / "constraints-3.6.txt", encoding="utf-8"
```

Comment:
Oh, BTW, we might not want to hardcode the `3.6` version, but instead use `UNIT_TEST_PYTHON_VERSIONS[0]`, i.e. the oldest one supported?

Reply:
It's a bit odd to use `UNIT_TEST_PYTHON_VERSIONS` in something that's a system test, but I agree it's worth having. I added a comment explaining why it's that version.